webapp/components/suggestion/at_mention_provider.jsx

```diff
@@ -112,9 +112,10 @@ export default class AtMentionProvider {
     handlePretextChanged(suggestionId, pretext) {
         clearTimeout(this.timeoutId);
 
-        const captured = (/@([a-z0-9\-\._]*)$/i).exec(pretext.toLowerCase());
-        if (captured) {
-            const prefix = captured[1];
+        const captured = (/(\w*)@([a-z0-9\-\._]*)$/i).exec(pretext.toLowerCase());
```

Comment:
Oh right. I forgot the first group is everything matched, even if the group isn't included. I used `/(^|\W)(@([a-z0-9.\-_]*))/` in text_formatting.jsx, but your way works just as well. Thanks for changing that!